runtime/doc/vim_diff.txt

```diff
@@ -50,7 +50,7 @@ the differences.
 - 'listchars' defaults to "tab:> ,trail:-,nbsp:+"
 - 'nrformats' defaults to "bin,hex"
 - 'ruler' is enabled
-- 'sessionoptions' excludes "options"
+- 'sessionoptions' excludes "options", includes "unix,slash"
```

Comment:
When was this added?

Reply:
@janlazo 2c1d12d0beda7b359fec94c00746b7206ae8fedd It's just something I noticed while working on https://github.com/neovim/neovim/pull/7836 and AFAICT there is no reason for platform differences here. This actually makes session files more portable, too. LMK if you disagree.

Reply:
I agree and thanks for the making the change.

Reply:
What about `mkview` and `viewoptions`? It supports `unix` and `slash`.

Reply:
Thanks, will update the doc for that also.